LongWarOfTheChosen/Localization/LW_Overhaul.ita

```diff
@@ -1,196 +1,202 @@
+;LWotC Translated
+
 [XComGameState_LWOverhaulOptions]
 LWOverhaulTabName="LONG WAR 2"
-GrazeBandWidthMod="Zona graffio"
+GrazeBandWidthMod="Zona Graffio"
 GrazeBandWidthModTooltip="Determina quanto deve essere vicino un colpo al centro esatto per contare come graffio"
 GrazeBandValue="Valore: <XGParam:IntValue0>"
-PauseOnRecruitMod="Pausa al reclutamento"
-PauseOnRecruitModTooltip="Pausa sul planisfero quando un rifugio recluta un nuovo soldato o un ribelle"
+PauseOnRecruitMod="Pausa al Reclutamento"
+PauseOnRecruitModTooltip="Pausa sul Planisfero quando un Rifugio recluta un nuovo soldato o un ribelle"
 
 [UIScreenListener_ShellDifficulty]
-strIncompatibleModsTitle="MOD DUPLICATI PRESENTI"
-strIncompatibleModsText="Rilevati mod duplicati:\n%modnames%\nQuesti mod sono stati integrati in questo mod e dovrebbero essere disabilitati dal launcher. Caricandolo potrebbero verificarsi effetti negativi, come scarse prestazioni o instabilità. Procedere a proprio rischio."
+strIncompatibleModsTitle="MOD DUPLICATE PRESENTI"
+strIncompatibleModsText="Rilevate mod duplicate:\n%modnames%\nQueste mod sono già state integrate in questa mod e dovrebbero essere disabilitate dal launcher. Caricandole potrebbero verificarsi effetti negativi, come scarse prestazioni o instabilità. Procedere a proprio rischio."
 
 [UIMission_LWCustomMission]
 m_strMissionDifficulty_start="ATTIVITÀ NEMICA BASE"
 
 [ProtectRegionEarly X2LWAlienActivityTemplate]
 ActivityName="ProtectRegion1Activity"
-ActivityObjectives[0]="Trova un indizio"
+ActivityObjectives[0]="Inizia a liberare la regione"
```

Comment:
So this is the line that needs to be warpped with
```; LWOTC Translated
...
; End Translated
```
because it's the one in _LW\_Overhaul.int_ that has `; LWOTC Needs Translation`. Does that make sense?